discord/channel.py

```diff
@@ -2053,6 +2055,263 @@ def listeners(self) -> list[Member]:
             member for member in self.members if member.voice and member.voice.suppress
         ]
 
+    async def _get_channel(self):
```

Comment:
Thats a big change here. Please explain

Reply:
Indeed I should have clarified; all `Messageable` channels have a `_get_channel` method to return the true destination, which is [used in `Messageable.send`](https://github.com/NeloBlivion/pycord/blob/stage_text/discord/abc.py#L1508) and several other functions. This has to be overridden, because by default [it is not implemented](https://github.com/NeloBlivion/pycord/blob/stage_text/discord/abc.py#L1310).

Reply:
Ah